app/Actions/Service/StartService.php

```
@@ -42,19 +42,37 @@ public function handle(Service $service, bool $pullLatestImages = false, bool $s
+        $compose = data_get($service, 'docker_compose', []);
+        $network = data_get($service, 'destination.network');
+
+        if (blank($network)) {
+            return [];
+        }
+
+        $commands = [];
+        $safeNetwork = escapeshellarg($network);
+        $serviceNames = data_get(Yaml::parse($compose), 'services', []);
+
+        foreach ($serviceNames as $serviceName => $serviceConfig) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$serviceConfig'. (undefined)

(UnusedLocalVariable)